### PR TITLE
Create FAQ page in sphinx User Guide

### LINKED ISSUE #1546: Consider creating an FAQ page in the User Guide
**Is your feature request related to a problem? Please describe.**
As with any software package with sufficiently large number of users, some usage questions get asked again and again.  Giving the askers the benefit of the doubt that they did some searching before posting their question, I will vouch that search engines are not always very useful if you don't know the magic terms of art to search for.  

**Describe the solution you'd like**
An FAQ page for things that get asked frequently but might not be found easily anywhere else in our docs.

Off the top of my head, here are some ideas for things it could cover:
- What is pvlib?
- How does pvlib compare to other tools (PVsyst, SAM, et al)
- All I have is GHI, how do I get to POA?
- Can I use PVsyst files (PAN/OND) files with pvlib?
- Why don't my simulation results make sense? (answer: check timezone localization)
- I got a warning like `RuntimeWarning: invalid value encountered in arccos`, what does it mean?
- [your suggestions here]

I'm sure trawling github issues/discussions, stackoverflow, and google group threads would turn up more good things to include.  

It might also be a good idea to explicitly try to speak "plain english" in FAQ entries, at least for the title of each entry.  For example an entry might be better titled "All I have is GHI, how do I get POA?" than something mentioning diffuse components and decomposition models. 

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by kanderso-nrel**
- [x] Closes #1546
 - [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - ~[ ] Tests added~
 - ~[ ] Updates entries in [`docs/sphinx/source/reference`](https://github.com/pvlib/pvlib-python/blob/master/docs/sphinx/source/reference) for API changes.~
 - [x] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/master/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - ~[ ] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.~
 - [x] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

Here's a first crack at an FAQ page.  Additions and revisions are very welcome.  I've marked it for 0.9.3 but also fine to push it to later.

Rendered page: https://pvlib-python--1549.org.readthedocs.build/en/1549/user_guide/faq.html

### CODE REVIEW

**Comment by AdamRJensen** [comment]
Hi @kanderso-nrel

pvlib sounds pretty cool! Is there anything I shouldn't use pvlib for or that it can't do? Financial calculations, uneven terrain?

File: docs/sphinx/source/user_guide/faq.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+
+How does pvlib compare to other PV modeling tools like PVsyst or SAM?
+---------------------------------------------------------------------
+
+pvlib is similar to tools like PVsyst and SAM in that it can be used
+for "weather-to-power" modeling to model system energy production
+based on system configuration and a weather dataset.  However, pvlib
+is also very different in that you use pvlib via python code instead
+of via a GUI.  pvlib is also more of a toolbox or a framework to use
```
Comment (AdamRJensen) [inline docs/sphinx/source/user_guide/faq.rst:27]:
```suggestion
of via a GUI, which makes pvlib ideal for automating tasks.  pvlib
is also more of a toolbox or a framework to use
```
Or something along those lines..

File: docs/sphinx/source/user_guide/faq.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+are available as well).
+
+
+Usage Questions
+***************
+
+All I have is GHI, how do I get to POA?
+---------------------------------------
+
+Going from GHI to POA irradiance is a two-step process.  The first step is to
```
Comment (AdamRJensen) [inline docs/sphinx/source/user_guide/faq.rst:38]:
```suggestion
Going from GHI to plane of array (POA) irradiance is a two-step process. 
The first step is to
```

**Comment by AdamRJensen** [comment]
**Where can I get irradiance data for my simulation?**
pvlib has a module called `iotools` which has several functions for retrieving irradiance data as well as reading standard file formats such as EPW, TMY2, and TMY3. For free irradiance data, you may consider NREL's NSRDB which can be accessed using the (:func:`pvlib.iotools.get_psm3` function and is available for North America. For Europe and Africa, you may consider looking into CAMS (:func:`pvlib.iotools.get_cams`). PVGIS (:func:`pvlib.iotools.get_pvgis_hourly`) is another option, which provides irradiance from several different databases with near global coverage. pvlib also has functions for accessing a plethora of ground-measured irradiance datasets, including the BSRN, SURFRAD, SRML, and NREL's MIDC.

**Comment by kanderso-nrel** [comment]
> pvlib sounds pretty cool!

Glad to hear you think so!  I agree :)

> Is there anything I shouldn't use pvlib for or that it can't do? Financial calculations, uneven terrain?

I'm a little hesitant to include a list of things that pvlib is unsuitable for.  For one, that list is infinitely long.  For another, what is unsuitable depends on the user -- for example I myself would surely turn to pvlib if I were tasked with modeling a system on uneven terrain, but I might not recommend others do the same.  I did try to implicitly address the financial calculations point by specifying _electrical_ performance in the "what is pvlib?" question.  

On the other hand, "how do I model variable terrain in pvlib?" has been asked before, so maybe that should be an item on its own?

**Comment by kanderso-nrel** [comment]
I have added a link to the README as @cwhanse suggested in https://github.com/pvlib/pvlib-python/issues/1546#issuecomment-1238636327.

Finding myself with no motivation to write an entry on variable terrain, I suggest we proceed with the current set of FAQ entries and expand the list in future PRs.

File: docs/sphinx/source/user_guide/faq.rst [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+and `two <https://groups.google.com/g/pvlib-python/c/b1mf4Y1qHBY/m/tK2FBCJyBgAJ>`_)
+may be useful for some users.
+
+
+Why don't my simulation results make sense? 
+-------------------------------------------
+
+pvlib does not prevent you from using models improperly and generating
+invalid results.  It is on you as the user to understand the models you
+are using.  However, one modeling error that beginners sometimes
```
Comment (cwhanse) [inline docs/sphinx/source/user_guide/faq.rst:82]:
```suggestion
invalid results.  It is on you as the user to understand the models you
are using and to supply appropriate, correctly-formatted data.  One modeling error that beginners sometimes
```

File: docs/sphinx/source/user_guide/faq.rst [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+run into is improper time zone localization: calculating solar
+positions is often the first step of a modeling process
+and relies on timestamps being localized to the correct time zone.
```
Comment (cwhanse) [inline docs/sphinx/source/user_guide/faq.rst:85]:
```suggestion
make is improper time zone localization. Calculating solar
positions is often the first step of a modeling process
and this step relies on timestamps being localized to the correct time zone.
```

File: docs/sphinx/source/user_guide/faq.rst [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+A telltale sign of improper time zones is a time shift between solar
+position and the irradiance data (for example, ``solar_elevation``
+peaks at a different time from clear-sky ``ghi``).
+For more information on handling timezone correctly, see :ref:`timetimezones`.
+
+More generally, inspecting the simulation results visually is a good first
+step when investigating strange results.
+Matplotlib and pandas have very powerful plotting capabilities that are great
+for tracking down where things went wrong in a modeling process.  Try plotting
+a few days of intermediate time series in a single plot, looking for
```
Comment (cwhanse) [inline docs/sphinx/source/user_guide/faq.rst:95]:
```suggestion
a few days of intermediate time series results in a single plot, looking for
```

**Review by cwhanse — APPROVED** [review]
Approving, some optional edits if you agree that they add clarification.

**Review by adriesse — APPROVED** [review]
Looks good to me.

**Pull request merged**